docs/microphone.rst

```diff
@@ -0,0 +1,109 @@
+Microphone **V2**
+*****************
+
+.. py:module:: microbit.microphone
+
+This object lets you access the built-in microphone available on the
+micro:bit **V2**. It can be used to respond to sound. The microphone input
+is located on the front of the board alongside a microphone activity LED,
+which is lit when the microphone is in use.
+
+.. image:: microphone.png
+    :width: 300px
+    :align: center
+    :height: 240px
+    :alt: micro:bit with microphone LED on
+
+Sound events
+============
+The microphone can respond to a pre-defined set of sound events that are
+based on the amplitude and wavelength of the sound. The ``SoundEvent`` class
+is already instantiated and can be used with the ``microphone`` methods
+documented below.
+
+Functions
```

Comment:
There is also an `is_event()` method: https://github.com/microbit-foundation/micropython-microbit-v2/issues/36

Reply:
Added in https://github.com/bbcmicrobit/micropython/pull/692/commits/8fe52d83b6eadf2dd74d94c71f4dbbc27c0ed779